{"name": "test_loginFailMethod", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <AppiumFramwork.tests.LoginTest.LoginTest testMethod=test_loginFailMethod>\n\n    @pytest.mark.run(order=1)\n    def test_loginFailMethod(self):\n        cl.allureLogs(\"App Opened\")\n        self.gt.clickLoginBotton()\n        self.gt.enterEmail()\n        self.gt.enterPassword2()\n        self.gt.clickOnLoginB()\n>       self.gt.verifyAdminScreen()\n\nLoginTest.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <AppiumFramwork.pages.LoginPage.LoginPageTest object at 0x00000271344FF8E0>\n\n    def verifyAdminScreen(self):\n        adminScreen = self.isDisplayed(self._pageTitle,\"text\")\n>       assert adminScreen == True\nE       AssertionError\n\n..\\pages\\LoginPage.py:44: AssertionError"}, "steps": [{"name": "App Opened", "status": "passed", "start": 1653559246867, "stop": 1653559246867}, {"name": "Click on Login Button", "status": "passed", "start": 1653559247626, "stop": 1653559247627}, {"name": "Entered email id", "status": "passed", "start": 1653559249260, "stop": 1653559249260}, {"name": "Entered Password", "status": "passed", "start": 1653559250346, "stop": 1653559250346}, {"name": "Clicked on Login Button in Login Screen", "status": "passed", "start": 1653559250923, "stop": 1653559250923}], "attachments": [{"name": "text", "source": "aa2570d6-d11f-4f95-b79d-e3ae0398568d-attachment.png", "type": "image/png"}, {"name": "text", "source": "b32c71c0-b523-456d-a19a-e3c31c194a53-attachment.png", "type": "image/png"}, {"name": "log", "source": "5d78c0dc-3418-4165-91ed-c9be4807719a-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "c56a7b0b-d656-44c5-86b4-211d60868828-attachment.txt", "type": "text/plain"}], "start": 1653559246865, "stop": 1653559277674, "uuid": "086f7f93-82f4-4e8a-bd05-63f447c37757", "historyId": "eb1ccc0d95544cdef7ee9a4e8770ec68", "testCaseId": "0e3892d464f1f164b14c4fe50287acd6", "fullName": "LoginTest.LoginTest#test_loginFailMethod", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "suite", "value": "LoginTest"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "BTZVBK3"}, {"name": "thread", "value": "8960-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "LoginTest"}]}